search_events_app/views.py

```diff
@@ -10,3 +11,12 @@ class EventListView(ListView):
 
     def get_queryset(self):
         return get_events()
+
+    def get_context_data(self, **kwargs):
+        countries = Country.objects.all()
+        context = super().get_context_data(**kwargs)
+        context['countries'] = countries
```

Comment:
are you using this in the template?

Reply:
Not yet, we are using countries_names, but we'll soon